packages/ui-components/src/ErrorDialogContent.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { caretDownIcon, caretRightIcon } from '@jupyterlab/ui-components';
+
+import * as React from 'react';
+
+const MESSAGE_DISPLAY = 'elyra-errorDialog-messageDisplay';
+const ERROR_DISPLAY_BUTTON = 'elyra-errorDialog-errDisplayButton';
+const ERROR_DETAILS = 'elyra-errorDialog-errDetails';
+const ERROR_DETAILS_VISIBLE = 'elyra-errorDialog-error-visible';
+const ERROR_DETAILS_HIDDEN = 'elyra-errorDialog-error-hidden';
+const ERROR_DIALOG_WIDTH = 600;
+const ERROR_DIALOG_HEIGHT = 400;
+const JP_DIALOG_CONTENT = 'jp-Dialog-content';
+
+interface IErrorDialogProps {
+  reason: string;
+  message: string;
+  timestamp: string;
+  traceback: string;
+  default_msg: string;
+}
+
+export class ErrorDialogContent extends React.Component<
```

Comment:
Is this more like an `ErrorDialogWidget` or `ErrorDialogComponent`

Reply:
I just kept the name, but following our other dialog body naming conventions we could change this to just `ErrorDialog`, `ServerErrorDialog`, or ExpandableErrorDialog` depending on how we want to classify it. I think the last one would be the most verbose and clear